tests/e2e/package-lock.json

```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@eclipse-che/api": {
-      "version": "7.40.1",
-      "resolved": "https://registry.npmjs.org/@eclipse-che/api/-/api-7.40.1.tgz",
-      "integrity": "sha512-2ij3D79RMcbV88XKbhI2fK3RgFex+qXDod4ZzXAx6LXoLHIdGTlYBs0EiCOmyuTYe2h4uB1tixRzGIVDTgc7PA=="
+      "version": "7.42.0",
```

Comment:
Let's keep 7.40.1 in here.

Reply:
It creates the conflict between package.json and pacakage-lock.json. do you want me to modify the @eclipse-che/api version from latest to 7.40.1

Reply:
yes, please

Reply:
modified the package-lock.json file as follows
```
    "version": "7.40.1",
      "resolved": "https://registry.npmjs.org/@eclipse-che/api/-/api-7.40.1.tgz",
      "integrity": "sha512-2ij3D79RMcbV88XKbhI2fK3RgFex+qXDod4ZzXAx6LXoLHIdGTlYBs0EiCOmyuTYe2h4uB1tixRzGIVDTgc7PA=="
```
Executed the command `npm ci`, looks like no issue, so pushing the same changes

Reply:
There is still `"@eclipse-che/api": "latest"` in package.json.